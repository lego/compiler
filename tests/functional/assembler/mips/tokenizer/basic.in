.word 1
.word 2
