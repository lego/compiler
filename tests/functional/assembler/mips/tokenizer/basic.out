DOTWORD .word 0 0
INT 1 1 6
DOTWORD .word 1 0
INT 2 1 6
